packages/cli/README.md

```diff
@@ -89,3 +89,16 @@ This project is built with [lerna](http://lernajs.io). The core plugins are loca
 To cut a release, simply run `lerna publish` and it will create a CHANGELOG from the pending commits using [Conventional Commits](http://conventionalcommits.org). CircleCI will run the jobs to publish the CLI once it receives the git tag.
 
 Review our [PR guidelines](./.github/PULL_REQUEST_TEMPLATE.md).
+
+Tips & Discoveries
+==================
+
+Below is an ongoing list of new discoveries and tips found through the nuances of the heroku cli.
+
+* Lerna tests will strip the color symbol from apps whereas the local tests per package don't. If you’re using @heroku-cli/color, remember to update your init.js file in your local helpers directory with
+   ```
+   // disable color for tests
+   const { color } = require('@heroku-cli/color')
+   color.enabled = false
+   ```
+   to strip the color. Otherwise, your tests will pass locally and fail lerna’s test
```

Comment:
If this is an issue should we just add the code to the repo to prevent others from running into this problem?